src/Infrastructure.EntityFramework/Repositories/DatabaseContextExtensions.cs

```diff
@@ -1,16 +1,20 @@
-﻿using Bit.Core.AdminConsole.Enums.Provider;
+﻿using System.Diagnostics;
+using Bit.Core.AdminConsole.Enums.Provider;
 using Bit.Core.Auth.Enums;
 using Bit.Core.Enums;
 using Bit.Infrastructure.EntityFramework.Repositories.Queries;
 using Microsoft.EntityFrameworkCore;
 
+#nullable enable
+
 namespace Bit.Infrastructure.EntityFramework.Repositories;
 
 public static class DatabaseContextExtensions
 {
     public static async Task UserBumpAccountRevisionDateAsync(this DatabaseContext context, Guid userId)
     {
         var user = await context.Users.FindAsync(userId);
+        Debug.Assert(user is not null);
```

Comment:
Question: I'm curious why we're using `Debug.Assert` here instead of doing `user!.AccountRevisionDate ...` like in some other places, presumably both options would throw an exception if `user` is null right?

Reply:
Honestly the other place in this PR _could_ probably use `Debug.Assert` as well. I treat `Debug.Assert`'s as my 0 cost way of making assumptions (and I should have used a better error message which I did change here). Basically the user id should be validated to actually exist in the database by some other way. The other one in this PR should probably actually handle null in that case because as far as I can tell it's a bit more possible.